apps/meteor/tests/end-to-end/api/25-teams.js

```diff
@@ -1570,6 +1572,71 @@ describe('[Teams]', () => {
 					.end(done);
 			});
 		});
+
+		describe('team auto-join', () => {
+			let testTeam;
+			let createdRoom;
+
+			beforeEach(async () => {
+				const createTeamPromise = createTeam(credentials, `test-team-name${Date.now()}`, 0);
+				const createRoomPromise = createRoom({ name: `test-room-name${Date.now()}`, type: 'c' });
+				const [testTeamCreationResult, testRoomCreationResult] = await Promise.all([createTeamPromise, createRoomPromise]);
+
+				testTeam = testTeamCreationResult;
+				createdRoom = testRoomCreationResult;
+
+				await request
+					.post(api('teams.addRooms'))
+					.set(credentials)
+					.expect(200)
+					.send({
+						rooms: [createdRoom.body.channel._id],
+						teamName: testTeam.body.team.name,
```

Comment:
```suggestion
						teamName: testTeam.name,
```
the `createTeam` helper already returns `body.team`, so this is not needed.